extensions/search-npm/src/index.tsx

```diff
@@ -6,16 +6,18 @@ import { NpmSearchFetchResponse } from './npmResponse.model'
 
 export default function PackageList() {
   const [searchTerm, setSearchTerm] = useState('')
+  const shouldFetch = searchTerm.trim() !== ''
   const { isLoading, data } = useFetch<NpmSearchFetchResponse>(
     `https://registry.npmjs.org/-/v1/search?text=${searchTerm.replace(
       /\s/g,
       '+',
     )}`,
+    { execute: shouldFetch },
   )
 
   return (
     <List
-      isLoading={isLoading}
+      isLoading={shouldFetch && isLoading}
```

Comment:
`isLoading` is set to `true` despite `execute` being set to `false`, it's no big deal, but I think `isLoading` should be `false` when `execute` is set to `false`? @mathieudutour 

(As a side note, I can't help but wonder: why are `@raycast/api` and `@raycast/utils` private?)

Reply:
That's fair! Thanks, looking forward to it :)

Reply:
Wonderful! I updated the PR to incorporate this change :)